e2e-tests/IIoTPlatform-E2E-Tests/Properties/launchSettings.json

```diff
@@ -1,9 +1,22 @@
 {
-  "profiles": {
-    "IIoTPlatform-E2E-Tests": {
-      "commandName": "Project",
-      "environmentVariables": {
-      }
-    }
-  }
+    "profiles":  {
+                     "IIoTPlatform-E2E-Tests":  {
+                                                    "commandName":  "Project",
+                                                    "environmentVariables":  {
```

Comment:
I think we don't need the environment variables here.

Reply:
yes, removed